tests/tests_pytorch/strategies/test_fsdp.py

```diff
@@ -142,6 +151,37 @@ def _assert_save_equality(trainer, ckpt_path, cls=TestFSDPModel):
             assert torch.equal(ddp_param.float().cpu(), shard_param)
 
 
+def custom_auto_wrap_policy(
+    module,
+    recurse,
+    unwrapped_params: int,
+    min_num_params: int = int(1e8),
+) -> bool:
+    return unwrapped_params >= 2
+
+
+def custom_auto_wrap_policy_2x(
```

Comment:
Nit: can we name it more explicitly like `_torch_gte_2_0`

Reply:
Kept the new conditional definition from [16764](https://github.com/Lightning-AI/lightning/pull/16764).